src/components/PopupBase/PopupBase.tsx

```diff
@@ -180,7 +184,7 @@ export const PopupBase: React.FC<Props> = ({
     return (
         <>
             {renderAnchor()}
-            {renderPopup()}
+            <div {...elem('popup')}>{renderPopup()}</div>
```

Comment:
If the renderPopup returns null the div will render anyway (check snapshots). Might be better to add the styling to the popup itself.